docs/_panels/gerrit-approvals.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Gerrit Approvals
+description: metrics focused on Gerrit approvals.
+author: Bitergia
+screenshot: sigils/gerrit-approvals.png
+created_at: 
+grimoirelab_version: 0.2.37
+layout: panel
+---
+
+This dashboard focuses on metrics derived from Gerrit approvals, not issued by bots. To this purpose, a filter 
+is set on the top of the dashboard (`NOT Bots`). The goal of the dashboard is to provide insights about the approvals in 
+terms of projects, repositories, reviewers and contributors. The visualizations composing the dashboard are described below.
```

Comment:
It might be better something like:

```
This dashboard focuses on metrics derived from Gerrit approvals. It provides insights in terms of projects, repositories, reviewers and contributors.
Due to automation is a key part in Gerrit workflow a filter is set on the top of the dashboard (`NOT Bots`) to ignore any kind of activity issued by bots. This is the default behaviour.

The visualizations composing the dashboard are described below.
```

 I think it is better to talk about `bots` after a general description of the dashboard.

Reply:
I don't see the change

Reply:
I tried to address your comment